foreman/foreman.spec

```diff
@@ -570,6 +588,8 @@ rm -rf %{buildroot}
 %config(noreplace) %{_sysconfdir}/logrotate.d/%{name}
 %config %{_sysconfdir}/cron.d/%{name}
 %{_sysconfdir}/rpm/macros.%{name}
+%{_sysconfdir}/rpm/macros.%{name}-apipie
+%{_sysconfdir}/rpm/macros.%{name}-assets
```

Comment:
You mean the %foreman_precompile_plugin macro should be romoved from -assets file? Should I move it to -plugin?